### ISSUE TITLE
[Comment] The Problem with C# 10 Implicit Usings

### ISSUE THREAD

**Issue by RehanSaeed** [RehanSaeed/rehansaeed.github.io#498]
https://rehansaeed.com/the-problem-with-csharp-10-implicit-usings/

**Comment by sam-wheat** [comment]
Another problem with implicit global usings is that because .net creates the globalusings file in the obj folder it does not create a file that you can modify as part of your project.  I think the correct behavior should be that .net creates a file for you that you can modify .  Also, if you use VS intellisense to add a using statement it adds the using to the current file only.  If you are using a GlobalUsings.cs file the using... statement will be added to the wrong file.

**Comment by teo-tsirpanis** [comment]
You shouldn't use the `System.Net.Http` NuGet package, [it's super old](https://www.nuget.org/packages/System.Net.Http#versions-body-tab). Instead you should add a `<Reference Include="System.Net.Http" />` item in your project; that references the built-in assembly in the framework.

**Comment by XtroTheArctic** [comment]
Can you please make the font of your blog website even bigger so it becomes totally unreadable instead of being partially unreadable?

**Comment by sam-wheat** [comment]
@XtroTheArctic It's not that the font is big - it's that you are small.

**Comment by XtroTheArctic** [comment]
Fair enough! I was joking you know. But still, your font is too big compared to the Internet standards :)

**Comment by XtroTheArctic** [comment]
Oh I just noticed your social buttons don't work. Just in case you want to fix them.
![image](https://github.com/RehanSaeed/rehansaeed.github.io/assets/6631429/1a3c213f-d73d-4102-ae8b-470347e094db)

**Comment by sam-wheat** [comment]
@XtroTheArctic Which page are you referring to - or the whole site?

**Comment by XtroTheArctic** [comment]
Originally, I was commenting on @RehanSaeed's blog website. Here is its big font size compared to CNN's text size which is also kinda big:
![image](https://github.com/RehanSaeed/rehansaeed.github.io/assets/6631429/3eadb2a4-9efd-45f5-8095-ea88358f017c)

After you @sam-wheat  joined the conversation, I checked your sites too:
![image](https://github.com/RehanSaeed/rehansaeed.github.io/assets/6631429/13860d26-7b6c-474a-9ee0-20c62ba6034d)
![image](https://github.com/RehanSaeed/rehansaeed.github.io/assets/6631429/74a37030-1440-4f43-bd59-7a9e318b53aa)
